controllers/licenseissuer/internal/manager/license_mgr.go

```diff
@@ -0,0 +1,337 @@
+/*
+Copyright 2023.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package manager
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"net/http"
+	"strconv"
+	"time"
+
+	"github.com/go-logr/logr"
+	accountv1 "github.com/labring/sealos/controllers/account/api/v1"
+	issuerv1 "github.com/labring/sealos/controllers/licenseissuer/api/v1"
+	"github.com/labring/sealos/controllers/pkg/crypto"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+const MaxSizeThresholdStr = "800Ki"
+
+const LicenseLifetime = time.Hour * 24 // 24 hours
+
+const (
+	CreatTimeField = "iat"
+	NodeField      = "nod"
+	CPUField       = "cpu"
+	DurationField  = "tte"
+	AddNodeField   = "and"
+	AddCPUField    = "adc"
```

Comment:
???

Reply:
This complies with the signature specification of asymmetric encryption